frontend/src/assets/css/variables.scss

```diff
@@ -12,5 +12,8 @@
   --blue: #3b79c3;
   --text-type-400: #130138;
   --text-type-300: #5a6474;
+  --card-widh: 294px;
+  --card-height: 296px;
+  --card-gap: 20px;
```

Comment:
That's nice to have reused things be moved to commons, but I believe `variables` should contain only global dependencies for styles -- so let's have this values declared locally for needed styles
```suggestion

```